src/Microsoft.AspNetCore.Mvc.Core/Filters/FilterCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Filters
+{
+    public static class FilterCollectionExtensions
+    {
+        public static IFilterMetadata Add<TFilterType>(this FilterCollection collection)
```

Comment:
No, these really should be instance methods. FilterCollection is specifically designed that way for it's use in options.

```
services.AddMvc(options =>
{
   options.Filters. // Oh, now I need to add a using....
});
```